application/gqlgen/watch.go

```diff
@@ -39,20 +39,46 @@ func (r *watchResolver) Owner(ctx context.Context, obj *models.Watch) (*PublicPr
 	return getPublicProfile(ctx, creator), nil
 }
 
-// Location resolves the `location` property of the watch query, retrieving the related record from the database.
-func (r *watchResolver) Location(ctx context.Context, obj *models.Watch) (*models.Location, error) {
+// Destination is a field resolver
+func (r *watchResolver) Destination(ctx context.Context, obj *models.Watch) (*models.Location, error) {
 	if obj == nil {
 		return &models.Location{}, nil
 	}
 
-	location, err := obj.GetLocation()
+	location, err := obj.GetDestination()
 	if err != nil {
-		return &models.Location{}, domain.ReportError(ctx, err, "GetWatchLocation")
+		return &models.Location{}, domain.ReportError(ctx, err, "GetWatchDestination")
```

Comment:
This should return `nil, ...` because Destination is nullable, right?
(The same would apply to lines 45, 59 and 64)

Reply:
Good catch, thanks.